core/account-id/src/lib.rs

```diff
@@ -425,6 +454,7 @@ mod tests {
         "b-o_w_e-n",
         "no_lols",
         "0123456789012345678901234567890123456789012345678901234567890123",
+        "b794f5eA0ba39494ce839613FFfba74279579268",
```

Comment:
In the NEP there is sometimes 0x prefix, can you double check if that is needed?